pyqtgraph/graphicsItems/AxisItem.py

```diff
@@ -1078,7 +1078,7 @@ def drawPicture(self, p, axisSpec, tickSpecs, textSpecs):
             p.setFont(self.tickFont)
         p.setPen(self.textPen())
         for rect, flags, text in textSpecs:
-            p.drawText(rect, flags, text)
+            p.drawText(rect, flags.__int__(), text)
```

Comment:
Just curious--is this somehow different from `int()`?

Reply:
i don't think so, I actually noticed this shortly after commenting above ... I'll test to make sure `int()` works, and update the PR.